install.sh

```diff
@@ -129,7 +130,7 @@ do
   # show usage & exit
   if [[ $arg == "-h" || $arg == "--help" ]];
   then
-    echo "Usage: bash install.sh [-h|--help] [-n|--nvidia {nx|nano|agx|tx2|raspi}] [-r|--robot {dingo|husky|jackal}] [-y|--yes]"
+    echo "Usage: bash install.sh [-h|--help] [-n|--nvidia {nx|nano|agx|tx2|raspi|amd64}] [-r|--robot {dingo|husky|jackal}] [-y|--yes]"
```

Comment:
`amd64` is ambiguous in this case. How about something along the lines of `desktop`?

Reply:
Changed. I had the opposite first impression that "desktop" was too ambiguous, which is why I went for the architecture name.